### ISSUE TITLE
unable to Undo

### ISSUE THREAD

**Issue by jgadsden** [OWASP/threat-dragon-core#4]
This issue has been migrated from : 
https://github.com/mike-goodwin/owasp-threat-dragon-core/issues/58
and was opened by @securestep9 : 

Undo command is missing in Edit - it makes life very difficult when you accidentally delete something by mistake (Redo is needed as well)

**Comment by leovdoo** [comment]
Although Undo and Redo labels are present in the Edit menu, they don't do anything.

**Comment by jgadsden** [comment]
Agreed @leovdoo - they only work when editing the project details and not when editing the diagram. The drawing package that we use, JointJS, does not support this for the free version; just the paid-for version 

We intend to move to mxgraph for version 2.0, and the project is here:
https://github.com/OWASP/threat-dragon-core/projects/6

with a roadmap here:
https://owasp.org/www-project-threat-dragon/#div-roadmap

If you would like to help then we are looking for contributors?
